From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for USB audio driver
Patch-mainline: Never, kABI workaround
References: bsc#1178203

The recent backport from 5.10 dropped the profile_name field from
snd_usb_audio_quirk, which broke kABI.  For the kABI compatibility,
restore the old filed again.  It's unused but no harm.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/usb/usbaudio.h |    1 +
 1 file changed, 1 insertion(+)

--- a/sound/usb/usbaudio.h
+++ b/sound/usb/usbaudio.h
@@ -107,6 +107,7 @@ enum quirk_type {
 struct snd_usb_audio_quirk {
 	const char *vendor_name;
 	const char *product_name;
+	const char *profile_name;	/* XXX: placeholder for kABI compatibility */
 	int16_t ifnum;
 	uint16_t type;
 	bool shares_media_device;
